community-handbook/metrics-release.md

```diff
@@ -2,27 +2,35 @@
+7. Address feedback in issue and through edits to the markdown page.
+8. Following comment period and edits, update the website to include the new metric. The website release will include the new metric with a label that it is ready for release. The link to the metric will point towards the working metric markdown page. Note: this means that the metric information displayed on the website may still be edited prior to the regular release.
+9. Create a release notes issue within the working group repository and update information for each new metric released. This issue will be used to create the regular release cadence release notes.
+
 ## Regular Release
 
-The regular release is when we update the version number and make a big announcement.
+The regular release is when we update the version number, update the full release notes, and make a big announcement. These releases occur one to two times a year and may correspond with the dates for CHAOSScon North America and Europe.
 
 1. Coordinate between working groups that a new release is coming.
 2. Each working group completes the “Continuous Metric Contribution” process for all of the metrics it wants to include in the release.
 3. On a set date, all metrics that successfully completed the “Continuous Metric Contribution” process will be included in the next release.
-4. Remove the “will be included in next release” statement on new metrics.
-5. The release notes will be updated to state that all changes are part of the new release.
-6. The release gets a new version number.
-7. Produce a PDF of the released metrics for archiving the metric release version.
-
-## Continuous Metric Contribution
-
-The goal is to have short cycles of feedback and to get metrics out sooner.
-
-1. Decide in a working group that a metric is ready for release.
-2. Merge metric into the working group repository.
-3. Include inside the metric that it will be included in next release.
-4. Create an issue for collecting feedback before release.
-5. Announce a 1-month feedback window on the mailing list and point to the issue.
-6. Address feedback.
-7. Update the website to include the new metric.
-8. Update the release notes to indicate that we changed the release.
+4. Remove the “will be included in next release” label on new metrics markdown pages and on the website.
+5. Create a GitHub release label (process is similar to creating a release branch) for all metrics that completed the “Continuous Metric Contribution”. This release label will replace the working markdown page the website was using during the continuous release period, creating a static version of the metric. The website will be edited to point to this static markdown page.
+6. The release notes will be updated - based on each of the working groups release issues - on the website to state that all changes are part of the new release. 
+7. The release gets a new version number.
```

Comment:
Item 7 should move up before item 5. We need the version number for the tagging.